v2.0/src/Components/Pages/StoriesPage/StoriesPage.js

```diff
@@ -330,13 +337,20 @@ class StoriesPage extends React.Component {
   // }
 
   renderStories(stories) {
+    if (!stories) {
```

Comment:
same thing here.  you actually need to check that the length of stories is not 0

Reply:
Same comment as above.  The case where no testimonials exist is returned from getContentToDisplay as null, where if there are no testimonials meeting the filter criteria, it is returned as an empty list